### PR TITLE
fix(docz-core): change filepath entry for windows

### LINKED ISSUE #12: Wrong path for components in Windows
## Bug Report

**Describe the bug**

When try to run dev project it return an error about wrong imports from components without the slash bar, probably something with windows path.

**To Reproduce**

1. Run `npm install`
2. Run `npm dev`

```shell
 ERROR  Failed to compile with 3 errors                                                                                                                                  09:20:48

These dependencies were not found:

* srccomponentsAlert.mdx in ./.docz/app/imports.js
* srccomponentsButton.mdx in ./.docz/app/imports.js
* srcindex.mdx in ./.docz/app/imports.js

To install them, you can run: npm install --save srccomponentsAlert.mdx srccomponentsButton.mdx srcindex.mdx
```

**Enviroment**

* OS: Windows 10
* Node/npm version: v10.4.0 / 6.0.0

**Additional context/Screenshots**
image

### LINKED ISSUE #30: Constantly getting missing dep on running on windows
## Bug Report

**Describe the bug**
If I make a MDX file I am getting this wierd missing dep error. I am using the typescript setup

**To Reproduce**
1. Install docz on windows
2. Make a MDX file and run docz

```shell
This dependency was not found:

* componentsButtonButton.mdx in ./.docz/app/imports.js

To install it, you can run: npm install --save componentsButtonButton.mdx
```
https://github.com/javascript-af/ui-lib
It is on the docz branch

**Expected behavior**

It should bootstrap the site and it should run 

**Enviroment**

* OS: Windows 10
* Node/npm version: Node 8.11.2

### PR DESCRIPTION
**Pull request by marceloavf**
### Description

Change Windows filepath imports

### Review

- [x] Check if continues working on mac/linux
- [x] Check if implementation is OK

Closes #12, Closes #30

### CODE REVIEW

File: packages/docz-core/src/Entries.tsx [hunk 1]
```diff
@@ -114,6 +114,9 @@ export class Entries {
     const entries = await Promise.all(files.filter(isEntry).map(createEntry))
 
     for (const entry of entries) {
+      if (process.platform === "win32") {
+        entry.filepath = entry.filepath.split('\\').join('/')
```
Comment (albinekb) [inline packages/docz-core/src/Entries.tsx]: I think you can use `path.normalize` https://nodejs.org/api/path.html#path_path_normalize_path

❯ node -p "require('path').win32.normalize('/Users/test')"
\Users\test

Comment (albinekb) [inline packages/docz-core/src/Entries.tsx]: I think you can use `path.normalize` https://nodejs.org/api/path.html#path_path_normalize_path

```
❯ node -p "require('path').win32.normalize('/Users/test')"
\Users\test

Comment (marceloavf) [reply packages/docz-core/src/Entries.tsx]: I tried with `path.normalize` and even `path.win32.normalize` but it doesn't work, strange 😞 

image

Comment (marceloavf) [reply packages/docz-core/src/Entries.tsx]: Seems that the tpl.js compiler just somehow mess up with the final import file with the double slash "\\" on windows

Comment (albinekb) [reply packages/docz-core/src/Entries.tsx]: https://github.com/pedronauck/docz/blob/master/packages/docz-core/templates/imports.tpl.js#L3

here?

Comment (marceloavf) [reply packages/docz-core/src/Entries.tsx]: Yes, it's all fine even on the last phase of writing the import, but after writing the import file the slash disappear on windows and the Error appear.

**Comment by pedronauck** [comment]
Thanks @marceloavf! You right in your pr, but I think that this need to be in the `Entry` class when define filepath of entry:

https://github.com/pedronauck/docz/blob/master/packages/docz-core/src/Entry.ts#L105-L108

**Comment by marceloavf** [comment]
Changed it @pedronauck, thanks!

**Review by pedronauck — APPROVED** [review]

**Pull request merged**